### PR TITLE
Use Rubocop to ensure consistently styled code.

### PR DESCRIPTION
**Pull request by mjgiarlo**
Add mandatory style-checking to the spec task. Clean up the README and badgify per Hydra convention.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
+[![API Docs](http://img.shields.io/badge/API-docs-blue.svg)](http://rubydoc.info/gems/hydra-works)
+[![Stories in Ready](https://badge.waffle.io/projecthydra-labs/hydra-works.png?source=projecthydra-labs%2Fhydra-works&label=ready&title=Ready)](https://waffle.io/projecthydra-labs/hydra-works?source=projecthydra-labs%2Fhydra-works)
 
-The Hydra::Works gem provides a set of [Portland Common Data Model](https://wiki.duraspace.org/display/FF/Portland+Common+Data+Model)-compliant models and associated behaviors around the broad concept of multi-file "works", the need for which was expressed by a variety of [community use cases](https://github.com/projecthydra-labs/hydra-works/tree/master/use-cases). The Hydra::Works domain model includes:
+The Hydra::Works gem provides a set of [Portland Common Data Model](https://wiki.duraspace.org/display/FF/Portland+Common+Data+Model)-compliant ActiveFedora models and associated behaviors around the broad concept of multi-file "works", the need for which was expressed by a variety of [community use cases](https://github.com/projecthydra-labs/hydra-works/tree/master/use-cases). The Hydra::Works domain model includes:
```
Comment (escowles) [inline README.md]: This should probably link to https://github.com/duraspace/pcdm/wiki instead of the Duraspace wiki.

Comment (mjgiarlo) [inline README.md]: Fixed, pushed!  Thx, @escowles

File: spec/hydra/works/models/collection_spec.rb [hunk 2]
```diff
@@ -77,63 +76,62 @@ class Cullection < Hydra::Works::Collection
...
       context 'that are unacceptable child collections' do
-
         let(:error_type1)    { ArgumentError }
         let(:error_message1) { /Hydra::Works::Generic(Work|File)::Base with ID:  was expected to works_collection\?, but it was false/ }
         let(:error_type2)    { NoMethodError }
         let(:error_message2) { /undefined method `works_collection\?' for .*/ }
 
-        it 'should NOT aggregate Hydra::Works::GenericWork in collections aggregation' do
-          expect{ subject.child_collections << @generic_work101 }.to raise_error(error_type1,error_message1)
+        it 'NOTs aggregate Hydra::Works::GenericWork in collections aggregation' do
```
Comment (escowles) [inline spec/hydra/works/models/collection_spec.rb]: This is pretty weird -- auto-rubocop change?

File: spec/hydra/works/models/collection_spec.rb [hunk 3]
```diff
@@ -231,58 +228,57 @@ class DummyExtWork < Hydra::Works::GenericWork::Base
...
       context 'that are unacceptable child generic works' do
-
         let(:error_type1)    { ArgumentError }
         let(:error_message1) { /Hydra::Works::(GenericFile::Base|Collection) with ID:  was expected to works_generic_work\?, but it was false/ }
         let(:error_type2)    { NoMethodError }
         let(:error_message2) { /undefined method `works_generic_work\?' for .*/ }
 
-        it 'should NOT aggregate Hydra::Works::Collection in generic works aggregation' do
-          expect{ subject.child_generic_works << @works_collection101 }.to raise_error(error_type1,error_message1)
+        it 'NOTs aggregate Hydra::Works::Collection in generic works aggregation' do
```
Comment (escowles) [inline spec/hydra/works/models/collection_spec.rb]: Another weird test wording change.

File: spec/hydra/works/models/collection_spec.rb [hunk 4]
```diff
@@ -399,24 +394,24 @@ class DummyExtWork < Hydra::Works::GenericWork::Base
       let(:non_PCDM_object)  { "I'm not a PCDM object" }
       let(:af_base_object)   { ActiveFedora::Base.new }
 
-      it 'should NOT aggregate Hydra::Works::Collection in related objects aggregation' do
-        expect{ subject.related_objects << collection1 }.to raise_error(ActiveFedora::AssociationTypeMismatch, /Hydra::Works::Collection:.*> is not a PCDM object./)
+      it 'NOTs aggregate Hydra::Works::Collection in related objects aggregation' do
```
Comment (escowles) [inline spec/hydra/works/models/collection_spec.rb]: Yet another weird wording change.

File: spec/hydra/works/models/collection_spec.rb (hunk 2 above)
Comment (mjgiarlo) [inline spec/hydra/works/models/collection_spec.rb]: Ugh, yes. The all-caps throws it off.  I'll grep for "NOT" and fix 'em all up.

File: spec/hydra/works/models/concerns/generic_file/virus_check_spec.rb [hunk 5]
```diff
@@ -7,8 +7,12 @@ class FileWithVirusCheck < ActiveFedora::Base
       include Hydra::Works::GenericFile::VirusCheck
     end
     class ClamAV
-      def self.instance; @instance ||= ClamAV.new; end
-      def scanfile(path); puts "scanfile: #{path}"; end
+      def self.instance
+        @instance ||= ClamAV.new
+      end
+      def scanfile(path)
+        puts "scanfile: #{path}"
```
Comment (escowles) [inline spec/hydra/works/models/concerns/generic_file/virus_check_spec.rb]: I think I put this in -- do we want to kill this puts as long as we're cleaning all this up?

Comment (mjgiarlo) [inline spec/hydra/works/models/concerns/generic_file/virus_check_spec.rb]: nope. nuking.

Comment (mjgiarlo) [inline spec/hydra/works/models/concerns/generic_file/virus_check_spec.rb]: actually, meh. this is fine. just spec-related.

**Pull request merged**